frontend/components/questionnaires/QuestionnaireForm.vue

```diff
@@ -14,106 +14,108 @@
+                                  {{ segmentScaleValue.value }} - {{ segmentScaleValue.text }}
+                                </li>
+                              </ul>
+                            </p>
+                          </div>
+
+                          <div v-if="segment.questions">
+                            <ul>
+                                <li 
+                                  v-for="(segmentQuestion, segQuIdx) in segment.questions" 
+                                  :ref="`question_${segQuIdx}`"
+                                  :key="`segmentQuestion-${segQuIdx}`">
+                                  <div v-if="segmentQuestion.id" class="question-container">
+                                    <p v-if="segment.prependIndex">
+                                      {{ segment.prependIndex+(segQuIdx+1) }}
+                                    </p>
+                                    <component 
+                                      :is="getComponent(segmentQuestion.type)"
+                                      v-model="segmentQuestion.value"
+                                      :header="segmentQuestion.header"
+                                      :required="segmentQuestion.required"
+                                      :question="segmentQuestion.text"
+                                      :options="segmentQuestion.options"
+                                      :config="segmentQuestion.config"
+                                      :is-clicked="segmentQuestion.isClicked"
+                                      :is-submitting="segmentQuestion.isSubmitting"
+                                      :error-message="segmentQuestion.errorMessage"
+                                      :read-only="segmentQuestion.readOnly"
+                                      :passed-data="{
+                                          question: segmentQuestion, 
+                                          formDataKey: getFormDataKey(segQuIdx, segIdx, qIdx)
+                                        }"
+                                      @change="onQuestionChange"
+                                    />
+                                  </div>
+                                  <p v-else>
+                                    This question doesnt contain proper information  
+                                  </p>
+                                </li>
+                            </ul>
+                          </div>
+                        </li>
+                      </ul>
+                      <!-- eslint-disable-next-line vue/no-v-html -->
+                      <footer v-html="questionnaire.footer" />
+                    </v-col>
+                  </v-row>
+                </v-card-text>
+                <v-card-actions>
+                  <v-btn v-if="activeQuestionnaire+1 < formData.questionnaires.length" 
+                    @click="onClickContinueButton">
+                    Continue to the next questionnaire 
```

Comment:
sorry, i totally forgot about this XD I have added all the translations now. Notably, I intentionally left the strings `This question doesnt contain proper information` and `not found` not translated because we will use these strings as indicators when something goes wrong (i.e. somehow the mapping suddenly does not match), so it is not necessary to translate them

Reply:
I personally think it should be translated to be consistent but alright XD

Reply:
Okay, I have added the translations 😅